wibotic_connector_can/src/wibotic_connector_can/wiboticros.py

```diff
@@ -31,9 +29,12 @@
 _shutting_down = False
 
 
-def ClearQueue(q):
-    while not q.empty():
-        q.get()
+def clear_queue(q):
+    try:
+        while True:
+            q.get_nowait()
+    except q.Empty:
+        pass
```

Comment:
The `queue.empty()` docstring states:

> Return True if the queue is empty, False otherwise (not reliable!).
> This method is likely to be removed at some point. 

The `queue.get_nowait()` ensures that blocking will not happen. This could happen in the case that an item was removed from the queue between the `queue.empty()` and `queue.get()`.